src/styles/index.scss

```diff
@@ -1,3 +1,153 @@
+$font-family-primary: 'Roboto', Arial, sans-serif;
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: 400px;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: 166px;
+    margin-inline: 16px;
+    margin-bottom: 16px;
+
+    &-title {
+      height: 36px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 1.5;
+      text-align: left;
+      color: #060b35;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    &-description {
+      margin: 0;
+      font-size: 10px;
+      line-height: 14px;
+      text-align: left;
+      color: #616070;
+    }
+  }
+
+  &__rating {
+    box-sizing: border-box;
+    width: 166px;
+    height: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-inline: 16px;
+    margin-bottom: 24px;
+
+    &-reviews {
+      font-size: 10px;
+      line-height: 14px;
+      text-align: right;
+      color: #060b35;
+    }
+  }
+
+  &__price {
+    box-sizing: border-box;
+    width: 166px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-inline: 16px;
+    margin-bottom: 16px;
+
+    &-title {
+      margin: 0;
+      font-size: 12px;
+      line-height: 18px;
+      text-align: left;
+      color: #616070;
+    }
+
+    &-cost {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      color: #060b35;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: #00acdc;
+    margin-inline: 16px;
+    margin-bottom: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    color: #fff;
+
+    &:hover {
+      box-sizing: border-box;
+      background-color: #fff;
+      border: 1px solid #00acdc;
+      color: #00acdc;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url('../images/star.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+
+    &:not(:last-child) {
```

Comment:
The `&:not(:last-child)` selector is unnecessary here because the `margin-right` is already set to 4px for all `.stars__star` elements, and it is overridden by this rule to the same value. You can safely remove this rule.